CHANGES.rst

```diff
@@ -1,3 +1,10 @@
+1.13.3 (unreleased)
+====================
+
+Bugfixes
+--------
+* Change the sequence of config file look up: current directory first, then user's home directory
```

Comment:
I think “Changes” not Change.  Also maybe “A config file in the working directory will now take precedence over a config file in the user’s home directory.” Vs the current phrasing